internal/pkg/client/request.go

```diff
@@ -94,6 +94,13 @@ func (r *Request) SetBody(body map[string]string) *Request {
 	return r
 }
 
+func (r *Request) SetJsonBody(body map[string]string) *Request {
```

Comment:
Může být takto? (Možná by spíš tohle mělo zůstat jako `SetBody` a ten druhý přejmenovat na `SetFormBody` když je to vlastně legacy, ale zase nechci v tomhle PR nadělat moc nepořádku. 🙂)

Reply:
ok 👍